### DISCUSSION TITLE
Your subscription has ended "COPILOT"

### DISCUSSION THREAD

**Discussion by HabunoGD1809** [community/community#40119] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have the same problem, I am a member "GitHub Student Developer Pack
"and I had my COPILOT working 100%. But it's been less than an hour since this has been causing problems.

![image](https://user-images.githubusercontent.com/68973717/204066284-2f3c9dda-347f-4047-aa15-032c53aa0ec9.png)

Anyone else with the same problem?

**Comment by jfullstackdev** [comment, marked as answer]
seems there are so many posts regarding this one, like this

https://github.com/orgs/community/discussions/40037

you can comment on there too, since a GitHub Staff already responded there

**Reply by HabunoGD1809** [reply]
Thank you
it is working perfect
